base/docs/helpdb/Base.jl

```diff
@@ -7994,7 +7994,10 @@ leading_ones
 """
     deserialize(stream)
 
-Read a value written by `serialize`.
+Read a value written by `serialize`. `deserialize` assumes the binary data read from stream is correct and has been serialized
```

Comment:
Wrap these at more like 80-92 chars. Either "data read from the stream" or quote `stream` as code to correspond with the input argument.